tests/core/full_node/test_mempool.py

```diff
@@ -1034,6 +1031,9 @@ def test_fun(coin_1: Coin, coin_2: Coin) -> SpendBundle:
 
         assert err is expected
         assert status == expected_included
+        if status == MempoolInclusionStatus.SUCCESS:
+            mempool_bundle = full_node_1.full_node.mempool_manager.get_spendbundle(bundle.name())
+            assert mempool_bundle is bundle
```

Comment:
this is what all the other test cases do. If `is` is more restrictive than `==` I wouldn't think this is a problem, and once it becomes a problem, it will be obvious and this can be changed

Reply:
`is` checks object identity vs. `==` checking equality.  Equailty is checked via `.__eq__()` so two instances of a class that are equal will pass `==` but not `is`.  This can get confusing especially with objects like integers and strings where Python has optimizations that will usually make two instances of the integer for two be the same object but two instances of value two thousand that are calculated separately won't pass `is` checks.  Similar with short vs. long strings.  But these are optimizations with confusing scenarios where the optimization does and does not happen.

Anyways, I'll accept consistency for now but my general policy is that `is` is appropriate for `None` and the vast majority of the time not for other objects.  And even with `None` it isn't really particularly useful, just such a prevalent practice that I stick with it.

Reply:
Along these lines: I believe `is` does pointer comparison, so it's a lot faster since it's guaranteed not to traverse down into an object and do recursive comparisons of contained objects. (Not that performance is likely a driving concern in this case.)